Resources/Private/Language/Search.xml

```diff
@@ -71,7 +71,7 @@
             <label index="tt_content.pi_flexform.collections">Suche auf diese Kollektionen einschränken</label>
             <label index="tt_content.pi_flexform.searchIn">Suche auf aktuelle/s Kollektion/Dokument einschränken?</label>
             <label index="tt_content.pi_flexform.searchIn.none">nein</label>
-            <label index="tt_content.pi_flexform.searchIn.document">nur Dokument</label>
+            <label index="tt_content.pi_flexform.searchIn.document">nur Dokument und Kindelemente</label>
```

Comment:
See above: "Kindelemente" implies that it works just for direct descendants but not for all.